src/nav/serverless-function-monitoring.yml

```diff
@@ -15,19 +15,21 @@ pages:
         pages:
           - title: Introduction to enabling Lambda monitoring
             path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-aws-lambda-monitoring
-          - title: 'Step 1: Link accounts'
+          - title: 'Link accounts'
             path: /docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/account-linking
-          - title: 'Step 2: Set up AWS API Polling or Metric Streams'
+          - title: 'Set up AWS API Polling or Metric Streams'
```

Comment:
I'd update the titles themselves in the doc to remove the "Step X" in all the docs

Reply:
done!